packages/botonic-api/src/data-provider/dynamodb-data-provider.ts

```diff
@@ -56,7 +57,15 @@ export class DynamoDBDataProvider implements DataProvider {
   }
 
   // @ts-ignore
-  async getUsers(limit = 10, offset = 0): Promise<User[]> {} // TODO: Implement
+  async getUsers(limit = 10, offset = 0): Promise<User[]> {
+    // TODO: offset really needed here?
+    const result = await this.userEventsTable.scan({
+      filters: { attr: SORT_KEY_NAME, beginsWith: USER_PREFIX },
+      limit,
+    })
+    if (result.Count === 0) return []
+    return result.Items
```

Comment:
If `result.Items` doesn't return a `[]` when it's an empty result, can't you go for a...
```
return result?.Items || []
```

Reply:
you were right, it just returns an empty list, returning it directly.